### DISCUSSION TITLE
Cannot access our account because email was never verified

### DISCUSSION THREAD

**Discussion by swimleftproducts** [deepgram/community#879] [General help] [answered]
Hi there, 

I have an account staging@leasly.ai that has api keys. We are actively using these api keys. However, I went to set up some new api keys and I found out I cannot access the account. When I went to reset the password, I was told I cant because the email was not verified. 

What can I do to use this account?

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by jkroll-deepgram** [comment, marked as answer]
Hi @swimleftproducts, I will reach out to that email, and if you reply to my email and confirm you would like to proceed, I will have a new verification email sent to you.

**Reply by swimleftproducts** [reply]
awesome. I just replied.
